app-frontend/package.json

```diff
@@ -23,6 +23,7 @@
     "test-watch": "karma start karma.conf.js --auto-watch --no-single-run"
   },
   "dependencies": {
+    "angucomplete-alt": "^3.0.0",
```

Comment:
do we really need an entire library for this? We aren't using a search API, and I expect that the number of classifications is most likely small enough that we could just request all of the classifiers in a project annotation from the api at once if we need to.